README.md

```diff
@@ -0,0 +1,176 @@
+# AWX Resource Operator
+
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Build Status](https://github.com/ansible/awx-resource-operator/workflows/CI/badge.svg?event=push)](https://github.com/ansible/awx-resource-operator/actions)
+[![Code of Conduct](https://img.shields.io/badge/code%20of%20conduct-Ansible-yellow.svg)](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) 
+[![AWX Mailing List](https://img.shields.io/badge/mailing%20list-AWX-orange.svg)](https://groups.google.com/g/awx-project)
+[![IRC Chat - #ansible-awx](https://img.shields.io/badge/IRC-%23ansible--awx-blueviolet.svg)](https://libera.chat)
+
+An kubernetes operator that can be used along side [Ansible AWX](https://github.com/ansible/awx) to launch automation jobs and create job templates by creating resources directly in kubernetes. This operator is built with [Operator SDK](https://github.com/operator-framework/operator-sdk) and Ansible.
+
+## Table of Contents
+
+* [AWX Resource Operator](#awx-resource-operator)
+   * [Overview](#overview)
+      * [Prerequisites](#prerequisites)
+   * [Installing the Operator](#installing-the-operator)
+   * [Usage](#usage)
+      * [AnsibleJob](#ansiblejob)
+      * [JobTemplate](#jobtemplate)
+   * [Building and Testing](#building-and-testing)
+   * [Author](#author)
+   * [Code of Conduct](#code-of-conduct)
+   * [Get Involved](#get-involved)
+
+## Overview
+
+There are currently two Custom Resources provided by the awx-resource-operator:
+
+* AnsibleJob - launches a job in the AWX instance specified in the the specified k8s secret (awx host url, token)
+* JobTemplate - creates a job template in the AWX instance specified.
+
+### Prerequisites
+
+* Install the kubernetes-based cluster of your choice:
+  * [Openshift](https://docs.openshift.com/container-platform/4.11/installing/index.html)
+  * [K8s](https://kubernetes.io/docs/setup/)
+  * [CodeReady containers](https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.5)
+  * [minikube](https://minikube.sigs.k8s.io/docs/start/)
+* Deploy AWX using the [awx-operator](https://github.com/ansible/awx-operator#basic-install)
+* Create an OAuth2 token for your user in the AWX UI
```

Comment:
Add steps for creating OAuth2 token

** Via the AWX UI:
  * Login to AWX with admin permissions
  * Navigate to Users
  * Select the username you wish to create a token for
  * Click on tokens, then the green plus icon
  * Application can be left empty, input a description and select a scope <r/w>